### ISSUE TITLE
Custom argument attributes not carried over

### ISSUE THREAD

**Issue by pfreitag** [foundeo/toscript#36]
Code such as:

    <cfargument name="controller" inject="coldbox">

Does not pass the `inject` argument over.

Could be converted as: 

    required any controller inject='coldbox'

Note that the type may be required in this case.

**Comment by KamasamaK** [comment]
I think it is fine, and personally preferred, to include the type when converting. I know this is beside the point, but the example conversion should not have `required`.

**Comment by pfreitag** [comment]
Yeah I think it is ok to add `any` in that case too, thanks for offering your opinion. I removed `required` from the original comment.

**Comment by KamasamaK** [comment]
Possibly related to #34
